ideascube/monitoring/tests/test_migration_unique_serial.py

```diff
@@ -0,0 +1,67 @@
+from ideascube.tests.helpers import migration_test
+
+
+@migration_test(
+    migrate_from=[('monitoring', '0005_auto_20161027_0801')],
+    migrate_to=[('monitoring', '0006_unique_serial')]
+)
+def test_do_not_break_already_unique_serial(migration):
+    Specimen = migration.old_apps.get_model('monitoring', 'Specimen')
+    StockItem = migration.old_apps.get_model('monitoring', 'StockItem')
+
+    stockItem = StockItem()
+
+    for i in range(5):
+        specimen = Specimen(item_id=stockItem.id, barcode=str(i), serial=str(i))
+
+    migration.run_migration()
+
+    Specimen = migration.new_apps.get_model('monitoring', 'Specimen')
+    specimens = Specimen.objects.order_by('barcode')
+    for specimen in specimens:
+        assert specimen.barcode == specimen.serial
+
+
+@migration_test(
+    migrate_from=[('monitoring', '0005_auto_20161027_0801')],
+    migrate_to=[('monitoring', '0006_unique_serial')]
+)
+def test_create_unique_serial(migration):
+    Specimen = migration.old_apps.get_model('monitoring', 'Specimen')
+    StockItem = migration.old_apps.get_model('monitoring', 'StockItem')
+
+    stockItem = StockItem()
```

Comment:
For some strange reason, my brain haven't make the connection between you `Dromedary case` and `camel case`. And for another strange reason, I thought you were saying me that the save was not needed. And as tests passed, it was ok for me.

I've readded the `.save()` and an extra (missing) `assert specimens.count() == 5` to be sure that we are really testing the value of the (saved) specimens.